dev/local-environment/configurations/cardano/cardano-node-2/config-pool2.json

```diff
@@ -0,0 +1,117 @@
+{
```

Comment:
This is same file as config-pool1.json
IMO should be moved to shared or maybe there is some ports customization missing here?